backend/fms_core/viewsets/_constants.py

```diff
@@ -242,6 +229,24 @@
     "project_name": CATEGORICAL_FILTERS_LOOSE,
     "metric_report_url": CATEGORICAL_FILTERS_LOOSE,
     "experiment_run": FK_FILTERS,
+    "readsets__validation_status": CATEGORICAL_FILTERS_LOOSE,
+    "readsets__release_status": CATEGORICAL_FILTERS_LOOSE,
+
+}
+
+_experiment_run_filterset_fields: FiltersetFields = {
+    "id": PK_FILTERS,
+    "name": CATEGORICAL_FILTERS_LOOSE,
+    "start_date": DATE_FILTERS,
+    "run_type": FK_FILTERS,
```

Comment:
If you break down the object in the filterset remove the FK filterset ... and make sure it is no longer referenced in the frontend...